jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/TaskCommonTest.java

```diff
@@ -41,14 +41,16 @@
   @Rule public final RestoreSystemProperties systemPropertyRestorer = new RestoreSystemProperties();
 
   @Mock private JibExtension jibExtension;
+  @Mock private ContainerParameters containerParameters;
   @Mock private Logger logger;
 
   @Before
   public void setUp() {
-    System.clearProperty("jib.extraDirectory.path");
-    System.clearProperty("jib.extraDirectory.permissions");
-    System.clearProperty("jib.extraDirectories.paths");
-    System.clearProperty("jib.extraDirectories.permissions");
+    Assert.assertNull(System.getProperty("jib.extraDirectory.path"));
+    Assert.assertNull(System.getProperty("jib.extraDirectory.permissions"));
+    Assert.assertNull(System.getProperty("jib.extraDirectories.paths"));
+    Assert.assertNull(System.getProperty("jib.extraDirectories.permissions"));
```

Comment:
I guess things got awry when merging with master. These should retain `clearProperty()`.